flang/docs/ArrayRepacking.md

```diff
@@ -400,7 +400,22 @@ Lowering of the new operations (after all the optimizations) might be done in a
 
 ### Runtime
 
-[TBD] define the runtime APIs.
+The array copies required for `pack/unpack` actions are done using `ShallowCopyDirect` API of flang-rt.
```

Comment:
I would add a note telling the motivation of doing so here, which as I understand it is performance since there is no need to copy the allocatable components data to make the top level entity contiguous, they can be moved to the repacked copy and moved back when unpacking.